src/api/index.tsx

```diff
@@ -28,8 +29,9 @@ export const callApi = async ({
   apiState,
   endpoint,
   method = HttpMethod.GET,
-  version = 'v4',
+  version = 'v4dev',
```

Comment:
Maybe we could also move this to a `REACT_APP_DEFAULT_API_VERSION` variable, WDYT? Given that it is done for the `lang` parameter, that would avoid an hardcoded value in the test file.